Board.java

```diff
@@ -44,7 +46,7 @@ public boolean checkForWin(char player) {
                if (count == 3) return true;
            }
         }
-        return checkForDraw();
+        return false;
     }
```

Comment:
the method name: checkForWin doesn't express that it is returning a boolean.  Really what it is doing is asking: 'is there a win? or have found a win?', maybe rename method so it is phrased like a question.